### PR TITLE
Added UDF documentation & examples

### PR DESCRIPTION
**Pull request by richard-rogers**
## Description

Notebook with UDF documentation and examples.

- [ ] I have reviewed the [Guidelines for Contributing](CONTRIBUTING.md) and the [Code of Conduct](CODE_OF_CONDUCT.md).

### CODE REVIEW

**Review by FelipeAdachi — COMMENTED** [review]
can you add it to examples/README? maybe under experimental, or advanced

**Review by FelipeAdachi — COMMENTED** [review]
Also, let's add the whylabs call to action and open in colab notebook button. When copy pasting the call to action, remember to edit the link to add the pointer to the current example

File: python/examples/experimental/whylogs_UDF_examples.ipynb
Comment (FelipeAdachi) [inline python/examples/experimental/whylogs_UDF_examples.ipynb:1]: maybe we could have the WARNING paragraph as a blockquote (starting with `>`)

**Review by jamie256 — APPROVED** [review]
LGTM! Can iterate on the follow up suggestions, lets get the new example in first.

**Pull request merged**